Development.md

```diff
@@ -57,6 +57,17 @@ To automatically fix issues run `npm run fix` command.
 ## Running Unit Tests
 Run `npm test` to execute the unit tests via [Jest](https://jestjs.io/).
 
+#### Unit Testing Guideline
+- Component tests should be located in the same directory as the tested component and named `component.spec.ts`.
+- Test utils and mocks are placed in `src/test`.
+- Focus on testing components that are shared across the project (see `src/app/shared`).
```

Comment:
I posted this comment but somehow it was in PENDING state-> reposting

Hi, Thanks for proposing i have opinion but i agree on some of the points:

IMHO,
Components which are isolated and pure based on Input parameters should be more **focus** on Unit Testing as said components which are `Shared` should be tested for Unit tests

 - Test should includes assertions for maximum combinations of assertions to be checked to validated truthiness of shared components rather than testing individual components.


 I am not sure but on Individual `*.spec.ts` i saw test cases like checking truthiness Which may or mayn't bring values to UI instead an extra overhead to UI IMHO.
 
 **Test cases like..**
  - Form is Valid or Not
  - Component is Initiated
  - Heading is checked or not
  - Existence of Element
  - Visibility of Element
  - Element is clickable or not
  ... etc

  I would say all above purpose can be handled in E2E cypress test cases that's all my personal opinion.
  
  **Note:**
  You can ignore it as it was discussed on TOI session with @maciaszczykm and @floreks might be chances we need to test it when checking template existence.